### ISSUE TITLE
[Refactoring] Using the ```Hash::make()``` instead of ```bcrypt()``` helper

### ISSUE THREAD

**Issue by simaremare** [Laravel-Backpack/community-forum#182]
# Feature Request
By default, Backpack uses the ```bcrypt()``` helper to generate the hashed password, I think this is an issue to flexibility since we can change the default ```bcrypt``` algorithm into something else in the ```config/hashing.php```. When changed to ```argon2id```, for example, Backpack breaks.

### What's the feature you think Backpack should have?
Backpack should use the ```Illuminate\Support\Facades\Hash::make()```.

### Have you already implemented a prototype solution, for your own project?
I replace all the ```bcrypt()``` to ```Hash::make()```.

### Do you see this as a core feature or an add-on?
The use of ```Hash::make()``` should be by default to promote higher flexibility.

**Comment by ziming** [comment]
Thought I share something I just found out today.

So, I have a task of trying to import users from internal App A to internal App B. App A is using argon2id, App B is using bcrypt

Initially I thought I am going to need to get everyone from App A to reset their passwords in App B, but to my surprise, I tried to login to App B with my account credentials in App A and it worked!

In the users table, that user record password is the argon2id hash (not the bcrypt hash), turns out laravel is smart enough to try both bcrypt and argon when verifying if a password is correct on authentication.

Then I go edit profile to update my password in App B, check the database, and it is now a bcrypt hash since App B hashing.php set bcrypt as the password hasher to use.

So this may not be a breaking change after all.
